### ISSUE TITLE
📚 Documentation: Fix broken links in Docs

### ISSUE THREAD

**Issue by divyanshugit** [NimbleEdge/EnvisEdge#170]
###  🙋 Where

List of broken links that need to be fixed:

- [ ] Contribute button on the side navigation bar. It should redirect to NimbleEdge/EnvisEdge.
- [ ] [Edit on GitHub on every page of docs](https://github.com/NimbleEdge/EnvisEdge/blob/master/docs/source/index.rst)
- [ ] [Dlrm_fl.yml on the getting started page](https://docs.nimbleedge.ai/configs/dlrm_fl.yml)
- [ ] In the tutorial section, the last line of the page mentions a link that should redirect to the next page. Example: [normal ML pipeline](https://docs.nimbleedge.ai/tutorials/Tutorial-Part-2-starting_with_nimbleedge.rst)
- [ ] Apart from these. Please do go through all the links and if you find do fix it.

### 💭 Description

There are a lot of broken links on the documentation page. Please go through all the links mentioned above and comment down if you need any sort of help or drop a message on Discord.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the guidlines?

- [X] I have read the Guildlines

### 🏢 Have you read the Code of Conduct?

- [X] I read the Code of Conduct

**Comment by Tob-iee** [comment]
Please can this task be assigned to me

**Comment by madhhuurrii** [comment]
Hii @divyanshugit, is this issue available?

**Comment by divyanshugit** [comment]
Sorry @madhhuurrii, Tobiee is already working on it. You can pick other [good first issues](https://github.com/NimbleEdge/EnvisEdge/labels/good%20first%20issue). If you need any suggestions/help, reach out to me over discord. I'll try my best to help you out.

**Comment by Dule-mart** [comment]
Those this issue still need help or @Tob-iee is still working on it?

**Comment by Tob-iee** [comment]
Yes, I'm still working on it, but apparently, there is a blocker. I have been able to talk to @ramesht007 about it. So he would have to get back to me on it before I proceed.

**Comment by Tob-iee** [comment]
I would like to continue working on this task @divyanshugit, @ramesht007  and @vkkhare, please can it be assigned to me. 

Also I would to ask we will consider going with trainer_modules or training_modules, As a new name for the experiments folders?

**Comment by Tob-iee** [comment]
And also I think some just re-created this as an issue a few hours ago, so I am a bit confused @ramesht007

**Comment by divyanshugit** [comment]
You can start working on this and make a PR. In the meantime, someone will assign this to you. On the second part, we can discuss it over Discord and decide.

**Comment by divyanshugit** [comment]
> And also I think some just re-created this as an issue a few hours ago, so I am a bit confused @ramesht007

That issue is specific to the README
